backend/infrahub/core/query/resource_manager.py

```diff
@@ -209,13 +213,23 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: dict[str, Any]) -> No
         self.params.update(branch_params)
 
         query = """
-        MATCH (pool:%(number_pool)s { uuid: $pool_id })-[r:IS_RESERVED]->(av:AttributeValue )
-        WHERE
-            toInteger(av.value) >= $start_range and toInteger(av.value) <= $end_range
-            AND
-            %(branch_filter)s
-        """ % {"branch_filter": branch_filter, "number_pool": InfrahubKind.NUMBERPOOL}
-
+        CALL {
```

Comment:
I think we need to add a MATCH statement before CALL ... not sure what is the expected behavior without it

Reply:
Thanks @ajtmccarty! 

>and it would be good to input attribute_name as a parameter so that the query can be cached

This is so that it would be treated as the same query instead of multiple different ones depending on which numberpool it was used against?

Reply:
that's correct